package exception;

public class M {

	public static void main(String[] args) {
		
	String name="name";
	try {
		int a=10;
		int b = 2;
		int c=a/b;;
		System.out.println(c);
		
	} catch (Exception e) {
		System.out.println("this is catch block");
	}
	
	System.out.println("swati");
		

	}

}
